examples/fluids/qfunctions/newtonian.h

```diff
@@ -15,6 +15,7 @@
 #include <math.h>
 #include <ceed.h>
 #include "newtonian_types.h"
+#include "newtonian_state.h"
```

Comment:
This commit should delete the now-duplicate code from this file. (It doesn't even compile due to redefinition errors.)

Reply:
Yep, I realized that soon after I pushed, but forgot to push the fix up last night. Somehow got dropped after moving my local branch from a custom `jed/fluids-jacobian`+`main` branch to `jed/fluid-jacobian` after you merged in `main`.